ape_trezor/client.py

```diff
@@ -102,80 +108,58 @@ def sign_personal_message(self, message: bytes) -> Tuple[int, bytes, bytes]:
         ethereum_message_signature = sign_message(
             self.client, self._account_hd_path.address_n, message
         )
-
         return extract_signature_vrs_bytes(signature_bytes=ethereum_message_signature.signature)
 
-    # TODO: Uncomment when Trezor has released the EIP 712 update
-    # def sign_typed_data(self, domain_hash: bytes, message_hash: bytes)
-    # -> Tuple[int, bytes, bytes]:
+    # def sign_typed_data(
```

Comment:
Is this still not ready yet?

Reply:
I think it is and was trying it out on separate branch but struggling to get it to work for some reason.
I think the bug in EIP-712 that recently got fixed may have been my issue though, so I am going to retry.

Because of the testing and such that goes along with it, that work will be on a separate PR.

But yes, you are correct - the methods we were waiting for are finally released to the public for third party wallets! So we should do that. GoodFirstIssue.

Reply:
alright, just thinking about the commented out code lol. would love to see it!